code/drasil-example/dblpend/lib/Drasil/DblPend/Requirements.hs

```diff
@@ -49,9 +50,7 @@ correct :: ConceptInstance
 correct = cic "correct" (foldlSent [
  atStartNP' (output_ `the_ofThePS` code), S "have the", 
  namedRef (propCorSol [] []) (plural propOfCorSol)]
- ) "Correct" nonFuncReqDom
+ ) "Correctness" nonFuncReqDom
 
 portable :: ConceptInstance
-portable = cic "portable" (foldlSent [
-  atStartNP (the code), S "is able to be run in different", plural environment])
-  "Portable" nonFuncReqDom
+portable = mkPortableNFR "portable" ["Windows", "Mac OSX", "Linux"] "Portability"
```

Comment:
At some point, having "established" OSes that can be selected is probably a better idea, but that could potentially wait for a future PR to be done after more discussion.